sqlx-mysql/src/options/parse.rs

```diff
@@ -78,6 +78,65 @@ impl MySqlConnectOptions {
 
         Ok(options)
     }
+
+    pub(crate) fn build_url(&self) -> Url {
+        let mut url = Url::parse(&format!(
+            "mysql://{}@{}:{}",
+            self.username, self.host, self.port
+        ))
+        .unwrap();
```

Comment:
`.unwrap()` in production code is bad form. Prefer `.expect()` with a grep-able context string:

```suggestion
        .expect("BUG: generated un-parseable URL");
```